source/source.go

```diff
@@ -37,6 +37,8 @@ const (
 	ttlAnnotationKey = "external-dns.alpha.kubernetes.io/ttl"
 	// The annotation used for switching to the alias record types e. g. AWS Alias records instead of a normal CNAME
 	aliasAnnotationKey = "external-dns.alpha.kubernetes.io/alias"
+	// The annotation used for defining whether externalIPs are ignored or not ( true / false )
+	ignoreExternalIPsAnnotationKey = "external-dns.alpha.kubernetes.io/ignoreExternalIPs"
```

Comment:
Just a suggestion: how about changing this to `external-dns.alpha.kubernetes.io/nodeport-target` with values `external-ip` or `internal-ip`? Looks more consistent to me.

Reply:
I like this suggestion as well

Reply:
+1 from me as well. Did some testing with the changes in this PR yesterday and it seems to work as expected so would be great to have this kind of functionality (be it this PR or #1310).